cicd/forgeops-tests/configure.py

```diff
@@ -0,0 +1,139 @@
+    current_dir = os.path.abspath(os.path.dirname(__file__))
+    os.chdir(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', '..'))
+    root_dir = os.getcwd()
+    os.chdir(current_dir)
+
+    helm_dir = os.path.join(root_dir, 'helm')
+    config_dir = os.path.join(root_dir, 'samples', 'config', TESTS_DEPLOYMENT)
+
+    env_components = "frconfig"
+    for component in TESTS_COMPONENTS.split(" "):
+        subfolder = eval('%s_HELM_SUBFOLDER' % component.upper())
+        image = eval('%s_IMAGE_NAME' % component.upper())
+        deps = eval('%s_DEPS_COMPONENTS' % component.upper())
+
+        values_yaml_file = os.path.join(helm_dir, subfolder, 'values.yaml')
+        # restore original file
+        if os.path.exists('%s.orig' % values_yaml_file):
+            shutil.copy2('%s.orig' % values_yaml_file, values_yaml_file)
+        # save file
+        shutil.copy2(values_yaml_file, '%s.orig' % values_yaml_file)
+
+        # read content of yaml file
+        values_yaml_content = ""
+        with open(values_yaml_file, 'r') as stream:
+            try:
+                values_yaml_content = yaml.load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
+
+        # update values
+        values_yaml_content['image']['repository'] = '%s/%s' % (IMAGE_BASE_URL, image)
+        values_yaml_content['image']['tag'] = TESTS_IMAGE_TAG
+        values_yaml_content['image']['pullPolicy'] = IMAGE_PULL_POLICY
+
+        # save new content
+        with open(values_yaml_file, 'w') as stream:
+            try:
+                yaml.dump(values_yaml_content, stream, default_flow_style=False)
+            except yaml.YAMLError as exc:
+                print(exc)
+
+        env_components = "%s %s %s" % (env_components, deps, component)
+
+    env_components = "%s web" % env_components
+
+    env_file = os.path.join(config_dir, 'env.sh')
+    # restore original file
+    if os.path.exists('%s.orig' % env_file):
+        shutil.copy2('%s.orig' % env_file, env_file)
+    # save file
+    shutil.copy2(env_file, '%s.orig' % env_file)
+
+    # read content of env file
+    env_content = ""
+    with open(env_file, 'r') as stream:
+        try:
+            env_content = stream.read()
+        except yaml.YAMLError as exc:
+            print(exc)
+
+    # update values
+    env_content = re.sub(r'NAMESPACE=.*', 'NAMESPACE=%s' % TESTS_NAMESPACE, env_content)
+    env_content = re.sub(r'DOMAIN=.*', 'DOMAIN="%s"' % TESTS_DOMAIN, env_content)
+    env_content = re.sub(r'COMPONENTS=.*', 'COMPONENTS=(%s)' % env_components, env_content)
+
+    # save new content
+    with open(env_file, 'w') as stream:
+        try:
+            stream.write(env_content)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+    print("TO DEPLOY: ../../bin/deploy.sh -n %s ../../samples/config/%s" % (TESTS_NAMESPACE, TESTS_DEPLOYMENT))
+    print("TO RUN TESTS: ./forgeops-tests.py tests/smoke -k 'test_idm and test_0_ping'")
```

Comment:
Does the line above need updating?

Reply:
Yes I have updated that and I have added some specific postcommit tests as discussed during the standup